MIGRATION.md

```diff
@@ -0,0 +1,35 @@
+# Guide for Migration from RHODS Codeflare-Operator to RHODS Data Science Cluster
+
+### Pre-requisites
+ - Existing installation of RHODS Codeflare-Operator installed via OperatorHub.
+
+## Migration Steps
+Follow these steps in order to migrate to using the updated RHODS Data Science Cluster:
+
+1. Uninstall ODH operator if you have it installed.
+    - In the [OpenShift Console](https://console.redhat.com/openshift/) navigate to Operators -> Installed Operators. Search for Open Data Hub, click on it and choose to uninstall.
+2. Uninstall RHODS CodeFlare Operator.
+     - In the [OpenShift Console](https://console.redhat.com/openshift/) navigate to Operators -> Installed Operators. Search for RHODS CodeFlare Operator, click on it and choose to uninstall.
+3. Install RHODS Version 2.4
+    - In the [OpenShift Console](https://console.redhat.com/openshift/) navigate to Operators -> OperatorHub. Search for Red Hat OpenShift Data Science and click on it. Choose version 2.4 and click Install.
```

Comment:
Currently I don't see version 2.4 in OperatorHub. I'm using QA OSD cluster. I did manage to get 2.4 via add-on though.